Src/Utils/ParseCsv.ts

```diff
@@ -32,8 +32,26 @@ export function parseCsv(data: string): [string[], number[][]]{
     return [schema, secrets];
 }
 
+function checkPieceSize(pieceSize: number) {
+    const lowerLimitSize: number = 1
+    const upperLimitSize: number = 1_000_000
+    if (pieceSize < lowerLimitSize) {
+        throw new Error(
+          "piece size " + pieceSize.toString() + " is over " +
+          "specified limit size " + lowerLimitSize.toString()
+        );
+    }
+    if (pieceSize > upperLimitSize) {
+        throw new Error(
+          "piece size " + pieceSize.toString() + " is under " +
+          "specified limit size " + upperLimitSize.toString()
```

Comment:
@mdonaka 
The error message "is under" and "is over" appear to be reversed.

Reply:
wow. That's right!
Thanks, fixed it with ade3b719a0250a6ce2897e9c44b8e13c10dca1ef.